src/renderer/components/ft-input-tags/ft-input-tags.vue

```diff
@@ -13,15 +20,21 @@
       :force-action-button-icon-name="['fas', 'arrow-right']"
       @click="updateTags"
     />
-
     <div class="ft-tag-box">
       <ul>
         <li
           v-for="tag in tagList"
           :key="tag.id"
         >
-          <span>{{ tag }}</span>
+          <img
+            v-if="tag.icon"
+            :src="tag.icon"
+            alt="icon"
```

Comment:
[This is not meaningful alt text.](https://www.howtogeek.com/821673/what-is-alt-text-why-you-should-use-it-and-how-to-write-it/) You should generally use an empty alt tag when the image's purpose is decorative.
```suggestion
            alt=""
```